.circleci/config.yml

```diff
@@ -105,7 +111,7 @@ jobs:
             CKE5_SLACK_NOTIFY_HIDE_AUTHOR: << parameters.hideAuthor >>
             CKE5_PIPELINE_NUMBER: << pipeline.number >>
           name: Waiting for other jobs to finish and sending notification on failure
-          command: yarn ckeditor5-dev-ci-circle-workflow-notifier
+          command: pnpm ckeditor5-dev-ci-circle-workflow-notifier
```

Comment:
The shell command should be executed using `pnpm exec <command>`, but I decided to skip the `exec` part, because:
- There is a very small chance of a name collision between the shell command name, the script name from `package.json`, and the built-in pnpm command name. Our binaries always have a long, unique names and are prefixed with `ckeditor5-*`.
- To keep the compatibility with other projects, where we execute shell commands via the shortcut `pnpm <command>` without the `exec` part.